src/WPF/WPF.Viewer/ArcGIS.WPF.Viewer.Net.csproj

```diff
@@ -20,6 +20,26 @@
     <NoWarn>1701;1702;CA1416</NoWarn>
   </PropertyGroup>
 
+  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|x86' OR '$(Configuration)|$(Platform)' == 'Release|x64' ">
```

Comment:
This could just say `Condition=" '$(Configuration)' == 'Release' "`

Reply:
You're right I just want to add it when publishing to the store, is there a way to work around this without swapping code in during the store package process?